doc/api/errors.md

```diff
@@ -2414,6 +2414,13 @@ error indicates that the idle loop has failed to stop.
 An attempt was made to use operations that can only be used when building
 V8 startup snapshot even though Node.js isn't building one.
 
+<a id="ERR_NOT_SUPPORTED_IN_SNAPSHOT"></a>
+
+### `ERR_NOT_SUPPORTED_IN_SNAPSHOT`
```

Comment:
What about using a more generic `ERR_NOT_SUPPORTED` error? The concrete description could be part of the message. We have many very specific error codes that likely do not help users all that much.

Reply:
I don't think a generic `ERR_NOT_SUPPORTED` would be quite useful in this case. What would one do after checking `error.code === ERR_NOT_SUPPORTED`? I can imagine the embedder put a TryCatch around e.g. LoadEnvironment and check for `error.code === ERR_NOT_SUPPORTED_IN_SNAPSHOT`, if that's true, simply give up creating a snapshot (as some kind of optional optimization) and then only do something with the raw code, otherwise simply treat it as a proper error that it cannot handle and fail. `ERR_NOT_SUPPORTED` would not be enough in this case, and then they'd end up..parsing the error message.

Reply:
Ping @BridgeAR does that answer the question?